### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by LappleApple**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,38 +1,31 @@
...
-A introduction on how it started:
-
-  https://tech.zalando.com/blog/monitoring-the-zalando-platform/
-
-Documentation:
-
-  http://zmon.readthedocs.org/en/latest/intro.html
+[Here's an introduction](https://tech.zalando.com/blog/monitoring-the-zalando-platform/) on how we started the project.
 
+[Here's more detailed documentation](http://zmon.readthedocs.org/en/latest/intro.html) than you'll find here.
```
Comment (hjacobs) [inline README.rst]: it uses "here" twice in the same sentence *wonder*

File: README.rst [hunk 2]
```diff
@@ -1,38 +1,31 @@
...
- * Check command and Alert condition are arbitrary Python expressions, giving you a lot of power
- * Store all results, incl. flattened dicts in time series in KairosDB
- * Internal charting and integrated Grafana for dash boards
- * Entity service to push entities, e.g. hosts, databases, ... we push ec2, elbs, ... via zmon-aws-agent
- * Define checks via yaml files and push using zmon-cli or via discovery from git sources
- * Use trial run in the UI to develop your checks/alerts with quick feedback
- * Trigger instant evaluation from UI for checks on longer intervals to rerun commands
-
-Start Demo using Vagrant
+ * Defines checks as data sources executed on self-defined entities
```
Comment (hjacobs) [inline README.rst]: "defines" is IMHO wrong here, as it's about YOU defining checks, not ZMON

The full sentence would be "You can define checks as data sources..."

File: README.rst [hunk 3]
```diff
@@ -1,38 +1,31 @@
...
+ * Defines alerts on checks and entities, with thresholds, as it suits your team
+ * Defines custom dashboards with widgets and alert filters based on teams and tags
+ * Check commands and Alert conditions are arbitrary Python expressions, giving you a lot of power
+ * Stores all results, including flattened dicts in time series, in KairosDB
+ * Offers internal charting and integrated Grafana for dashboards
+ * Entity service to push entities: hosts, databases, et al. We push EC2, ELB, etc. via [zmon-aws-agent](https://github.com/zalando/zmon-aws-agent)
+ * Defines checks via YAML files and pushes using [zmon-cli](https://github.com/zalando/zmon-cli) or via discovery from Git sources
+ * Uses trial run in the UI to develop your checks/alerts with quick feedback
```
Comment (hjacobs) [inline README.rst]: Same here as: the user uses trial run

File: README.rst [hunk 4]
```diff
@@ -1,38 +1,31 @@
...
- * Define checks via yaml files and push using zmon-cli or via discovery from git sources
- * Use trial run in the UI to develop your checks/alerts with quick feedback
- * Trigger instant evaluation from UI for checks on longer intervals to rerun commands
-
-Start Demo using Vagrant
+ * Defines checks as data sources executed on self-defined entities
+ * Defines alerts on checks and entities, with thresholds, as it suits your team
+ * Defines custom dashboards with widgets and alert filters based on teams and tags
+ * Check commands and Alert conditions are arbitrary Python expressions, giving you a lot of power
+ * Stores all results, including flattened dicts in time series, in KairosDB
```
Comment (hjacobs) [inline README.rst]: "ZMON stores all results ...", i.e. here it's ok

File: README.rst (hunk 1 above)
Comment (LappleApple) [inline README.rst]: for the second "here," you can put "than you'll find in this README."

File: README.rst [hunk 5]
```diff
@@ -67,11 +60,11 @@ KairosDB frontend, i.e. for manually query of metrics:
 Issues
 ------
 
-* every LDAP user has admin role (because I could not get the OpenLDAP "memberOf" overlay to work
+* Every LDAP user has an admin role, because we could not get the OpenLDAP "memberOf" overlay to work.
```
Comment (hjacobs) [inline README.rst]: "the" admin role? (there is only one)

File: README.rst [hunk 6]
```diff
@@ -67,11 +60,11 @@ KairosDB frontend, i.e. for manually query of metrics:
...
 
-* If single containers do not start up ssh into the vagrant box and run the start.sh script again manually or use the start-services.sh script to restart single components. Later one takes parameters like controller or worker.
+* If single containers do not start up SSH into the Vagrant box and run the start.sh script again manually, or use the start-services.sh script to restart single components, later ones take parameters like controller or worker.
```
Comment (hjacobs) [inline README.rst]: "ssh" is used as a verb here

**Comment by hjacobs** [comment]
@LappleApple thanks for the PR, see my comments :smile:

File: README.rst (hunk 2 above)
Comment (LappleApple) [inline README.rst]: OK. I was trying to achieve grammar-related consistency among the bullet points (ie make them all start with verbs).

File: README.rst (hunk 5 above)
Comment (LappleApple) [inline README.rst]: Sure, if only one admin then "the" fits.

File: README.rst (hunk 6 above)
Comment (LappleApple) [inline README.rst]: Really? Then the verb = "do not start up SSH" (mindblown gif)

Comment (hjacobs) [inline README.rst]: probably a comma or "then" is missing:

"If ... do not start up then ssh into ..."

Comment (LappleApple) [inline README.rst]: That sounds right. + comma after "up" and it makes sense

**Comment by hjacobs** [comment]
@LappleApple I merged your proposed changes manually into the master branch, therefore I'm closing this PR now.

**Pull request closed without merging**